traffic_ops/install/bin/_postinstall

```diff
@@ -569,7 +569,7 @@ sub getDefaults {
                 "config_var"      => "tm.url"
             },
             {
-                "Human-readable CDN Name.  (No whitespace, please)" => "kabletown_cdn",
+                "Human-readable CDN Name.  (No whitespace, please)" => "kabletownCdn",
```

Comment:
So my only worry about this is that if underscores really only break DNS-related fields, shouldn't we just target those fields more specifically and make sure they're valid? `cdn_name` should be ok to have underscores in it I'd think.

Reply:
I was matching the regex that is accepted by the ops api. Per our discussion however it seems that there is a risk of breaking existing testing suites etc by restricting the use of underscores. I will change the regex to accept underscores and leave the sub for adding additional checks on other fields.